glpi/glpi.py

```diff
@@ -115,7 +117,7 @@ class GlpiService(object):
 
     def __init__(self, url_apirest, token_app, uri=None,
                  username=None, password=None, token_auth=None,
-                 use_vcap_services=False, vcap_services_name=None):
+                 use_vcap_services=False, vcap_services_name=None, sslverify=True, writable = False):
```

Comment:
Why you are enabling by default the SSL Verifying? What is the default behavior when you have an self signed certificate?